index.html

```diff
@@ -3066,8 +3092,11 @@ <h5>JWT Encoding</h5>
 JOSE header parameters, JWT registered <a>claim</a> names, or contained in the
```

Comment:
I'm not sure I understand the three possibilities in the MUST. I *think* "encoded as standard JOSE header parameters, JWT registered" -> "encoded as standard JOSE header parameters, encoded as JWT registered" but if that's not what was meant, then some other word-smithing is needed.

Reply:
This comment refers to text not altered by this pull request. If there are additional changes to be executed here, they should be addressed in a separate commit.

Reply:
OK.  https://github.com/w3c/vc-data-model/issues/552